src/App.scss

```
@@ -1 +1,21 @@
 // styles go here
+h1 {
+margin: 0;
+padding: 20px;
+color: rgb(77, 75, 75);
+text-align: center;
+}
+
+.App {
```

```suggestion
.App {
```
Назви з великої літери використовуються для назв компонентів. Для класів краще підібрати іншу назву (з маленької літери)